src/common/CvtFormat.cpp

```diff
@@ -75,7 +75,7 @@ namespace
 			const TrieNode* currentNode = m_root;
 			FB_SIZE_T valueLength = fb_strlen(value);
 
-			for (outParsedTimezoneLength = 0; outParsedTimezoneLength < valueLength; outParsedTimezoneLength++)
+			for (outParsedTimezoneLength = 0; static_cast<FB_SIZE_T>(outParsedTimezoneLength) < valueLength; outParsedTimezoneLength++)
```

Comment:
I think it's not the right way to remove the warning, we are just replacing implicit cast to explicit. I think it would be better to change type of `outParsedTimezoneLength` to FB_SIZE_T, so we can be 100% sure about this comparison. Yeah, we would have to change more code, but I think this is the right approach to handle warnings.